tests/manual-test-cases/Group19-ROBO/19-1-ROBO-SKU.md

```diff
@@ -15,10 +15,10 @@ This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter
 2. Add the ROBO SKU license to the vCenter appliance
 3. Assign the ROBO SKU license to each of the hosts within the vCenter
 4. Install the VIC appliance onto one of the hosts in the vCenter
```

Comment:
Think this should be explicit about standalone hosts, single host clusters, or multi-host clusters.